packages/inventory/src/containers/ProfileView/utils.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ColumnsType, ColumnType } from 'antd/lib/table/interface';
+import { INVENTORY_SERVICE_POINT_PROFILE_VIEW, TableColumnsNamespace } from '../../constants';
+import { Link } from 'react-router-dom';
+import { LocationUnit, TreeNode } from '@opensrp/location-management';
+import {
+  ACTIONS_TH,
+  PRODUCT_NAME_TH,
+  QTY_TH,
+  PO_NUMBER_TH,
+  SERIAL_NUMBER_TH,
+  DELIVERY_DT_TH,
+  ACCOUNT_END_DT_TH,
+  UNICEF_SECTION_TH,
+  DONOR_TH,
+} from '../../lang';
+import { GeographicLocationInterface } from '.';
```

Comment:
should we move `GeographicLocationInterface` here to avoid circular-ish imports

Reply:
`GeographicLocationInterface` is defined once in profile index page, So imported from there instead of declaring same thing twice.